npm-packages/meteor-installer/install.js

```diff
@@ -1,4 +1,5 @@
 const { DownloaderHelper } = require('node-downloader-helper');
+const HttpsProxyAgent = require('https-proxy-agent');
```

Comment:
Yes you should! I’m not around my pc now but I will ask @denihs if he can make a new installer release.
I do have the package globally and did not realized that it may be needed. My bad.